### PR TITLE
clarify steps, md best practices

### PR DESCRIPTION
**Pull request by timroster**

### CODE REVIEW

File: tutorial/Lab1/README.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-# Deploy the application using `kubectl`
+## Scenario 1: Deploy the application using `kubectl`
 
-In this part of the lab, we will deploy the application using the Kubernetes client `kubectl`. We will use [Version 1](https://github.com/IBM/guestbook/tree/master/v1) of the app for deploying here. Clone the [Guestbook App](https://github.com/IBM/guestbook) repo to get the files: 
-```$ git clone https://github.com/IBM/guestbook.git``` .
+In this part of the lab, we will deploy the application using the Kubernetes client `kubectl`. We will use [Version 1](https://github.com/IBM/guestbook/tree/master/v1) of the app for deploying here. Clone the [Guestbook App](https://github.com/IBM/guestbook) repo to get the files:
+
+```console
+git clone https://github.com/IBM/guestbook.git
+```
```
Comment (lee-zhg) [inline tutorial/Lab1/README.md:13]: We need to move to the right folder before running k8s commands. So, I have

```
$ cd guestbook/v1
```

File: tutorial/Lab1/README.md [hunk 2]
```diff
@@ -69,101 +74,135 @@ In this part of the lab, we will deploy the application using the Kubernetes cli
...
 
-# Deploy the application using Helm
+## Scenario 2: Deploy the application using Helm
 
 In this part of the lab, we will deploy the application by using Helm. We will set a release name of `guestbook-demo` to distinguish it from the previous deployment. The Helm chart is available [here](../../charts/guestbook). Clone the [Helm 101](https://github.com/IBM/helm101) repo to get the files:
-```$ git clone https://github.com/IBM/helm101``` .
+
+```console
+git clone https://github.com/IBM/helm101
+```
```
Comment (lee-zhg) [inline tutorial/Lab1/README.md:83]: Moved to right folder before Helm command. I added

```
cd helm101/charts
````

File: tutorial/Lab1/README.md (hunk 1 above)
Comment (timroster) [reply tutorial/Lab1/README.md:13]: The cd command is here: https://github.com/IBMAppModernization/helm101/blame/231b11618c1c79cb6b7e4f68165385f9b0c16842/tutorial/Lab1/README.md#L18 

that way it is part of the steps.

File: tutorial/Lab1/README.md (hunk 2 above)
Comment (timroster) [reply tutorial/Lab1/README.md:83]: The cd command is here:

https://github.com/IBMAppModernization/helm101/blame/231b11618c1c79cb6b7e4f68165385f9b0c16842/tutorial/Lab1/README.md#L115

**Pull request merged**